test-files/io/ServiceHootApiDbReaderTest/runReadByBoundsTestInput.osm

```diff
@@ -1,78 +1,78 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
-    <node visible="true" id="1" timestamp="1970-01-01T00:00:00Z" lat="38.9008975" lon="-78.0226543">
+    <node visible="true" version="1" id="1" timestamp="1970-01-01T00:00:00Z" lat="38.9008975" lon="-78.0226543">
```

Comment:
Changing the input here to include the `version` attribute so that it can be loaded and visualized in JOSM.